modules/streaming/src/main/scala/io/circe/streaming/ParsingConfiguration.scala

```diff
@@ -0,0 +1,11 @@
+package io.circe.streaming
+
+import jawn.AsyncParser
+
+case class ParsingConfiguration(parseMode: AsyncParser.Mode)
+
+object ParsingConfiguration {
+
+  implicit val defaultParsingConfiguration: ParsingConfiguration =
```

Comment:
@vkostyukov I'm afraid it's not possible to distinguish `SingleValue` and `StreamValue` modes easily